README.ja.md

```diff
@@ -254,7 +254,9 @@ Server: WebSocket++/0.7.0
 
 この例を実行するには、`https://localhost:8443 `で WebRTC 互換ブラウザ を開き、ページをオープンするために、HTTPS トラフィックを受け入れます。アプリケーションは、2つの HTML5  `<video>` タグを含む、1つの HTML Web ページで構成されています。1つはローカル Web-cam でキャプチャされたローカル・ストリームと、もう1つはメディア・サーバからクライアントに送信されたリモート・ストリームです。"Start" ボタンをクリックすると、両方の `<video>` 要素に同じビデオが表示されます。
 
-![](https://fiware.github.io/tutorials.Media-Streams/img/hello-world-screenshot.png)
+[![](https://fiware.github.io/tutorials.Media-Streams/img/hello-world-screenshot.png)](https://www.youtube.com/watch?v=vGEnkSOp_xc "Hello World")
```

Comment:
In the end I didn't go for a link on the image directly, could you revert this line back to a static image please?